### ISSUE TITLE
ControlStructureSpacing reports false positive with comments

### ISSUE THREAD

**Issue by alcaeus** [slevomat/coding-standard#532]
For the following code:
```php
        }

        // This is a longer comment that
        // spans across multiple lines
        return count(array_keys($context->getCurrentPath(), 'similarProducts')) >= 1;
```

the following error is reported:
```
Expected 1 lines before "return", found 2.
(SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure)
```

The error is marked as fixable but can't be fixed, probably because of the comment. I'd expect no error to be reported here.

**Comment by kukulich** [comment]
Duplicate of #520

**Comment by alcaeus** [comment]
Dang, I searched for issues but apparently not the right ones. Sorry for the noise.

**Comment by kukulich** [comment]
No problem :) I'm happy I don't any new issue :D
